lib/mutant/mutator/node/arguments.rb

```diff
@@ -36,7 +36,12 @@ def only_mlhs?(new_arguments)
         end
 
         def forward_arg?
-          children.last && n_forward_arg?(children.last)
+          children.any?(&method(:forward_type?))
+        end
+
+        def forward_type?(node)
+          n_forward_arg?(node) ||
+              ((n_restarg?(node) || n_kwrestarg?(node)) && node.children.empty?)
```

Comment:
@mbj This isn't as strict as it could theoretically be. I believe this will remove mutations like:

`def foo(*); end` -> `def foo; end` (and the same for kwargs) even if there is no argument forwarding. I think this is fine. The alternative would be to recurse into the body and look for references to the arguments being forwarded.